src/installer/Launcher.bat

```diff
@@ -2,15 +2,12 @@
 
 set ORIGINAL_DIR=%CD%
 
-:: This command gets the location of the directory that holds this script.  We
-:: use this to set up the rest of the paths for launching.  If you want to copy
-:: this script elsewhere, you should modify BPHOMEDIR to explicitly set the 
-:: location where BridgePoint is installed to.
-set BPHOMEDIR=%~dp0%
+:: BPHOMEDIR is used at runtime to determine the base installation folder.
+:: It is the BridgePoint folder.
+cd "%~dp0%/.."
+set BPHOMEDIR=%CD%
 
-::
-:: DO NOT MODIFY ANY OF THE FOLLOWING LINES.
-::
+:: The JVM tested with BridgePoint
 set BP_JVM=%BPHOMEDIR%/../jre/bin/javaw.exe
```

Comment:
I thought BPHOMEDIR now stopped at "BridgePoint" and not "BridgePoint/eclipse".  If it does stop at "BridgePoint" like the linux version then this line (11) and the following references (14 and 18) should not have "../" in the path.  And line 21 should change to "cd %BPHOMEDIR%/eclipse"